src/screens/FeedScreen/ContentLink.js

```diff
@@ -105,7 +118,7 @@ const ContentLink = ({
             topics={topics}
             fontSize={normalizeFontSize(14)}
             text={sanitizeUrl}
-            topicContainer={styles.topicStyle}
+            topicContainer={styles.topicStyle(item?.images_url?.length > 0)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `topicStyle` function now accepts a parameter to adjust styling based on the presence of images. This is a good approach to dynamically adjust styles based on the content.
```diff
+ topicContainer={styles.topicStyle(item?.images_url?.length > 0)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->